app/services/bulk_tagging/links_publisher.rb

```diff
@@ -14,12 +14,24 @@ def publish
       if links_update.valid?
         Services.publishing_api.patch_links(
```

Comment:
Perhaps we should use the publishing-api versioning system here.

The `publishing_api#get_links` call returns a `version`, which you can pass into `patch_links`. 

If somebody else modified the links in the meantime, publishing-api will return a HTTP Conflict status code (raises an exception). This prevents accidental overriding of data.